lib/widgets/resources/arm.dart

```diff
@@ -0,0 +1,280 @@
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:viam_sdk/viam_sdk.dart';
+import 'package:viam_sdk/widgets.dart';
+
+/// A widget to control an [Arm].
+class ViamArmWidget extends StatefulWidget {
+  /// The [Arm]
+  final Arm arm;
+
+  const ViamArmWidget({
+    Key? key,
+    required this.arm,
+  }) : super(key: key);
+
+  @override
+  State<ViamArmWidget> createState() => _ViamArmWidgetState();
+}
+
+class _ViamArmWidgetState extends State<ViamArmWidget> {
+  Pose endPosition = Pose();
+  List<double> jointPositions = [];
+
+  Future<void> _positions() async {
```

Comment:
[minor] I think in this case it is okay to prefix the method name with `get`.  omitting `get` / `set` is more for API design, since this the internal workings of a widget and isn't a public API I think naming the method more clearly would be helpful.

```suggestion
  Future<void> _getPositions() async {
```

ref: https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get